modules/oadp-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,231 @@
+.Example
+[source,yaml]
+----
+spec:
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+# ...
+----
+
+* The `spec.configuration.restic` field is being deprecated in OADP 1.3, and will be removed in OADP 1.4. To avoid seeing deprecating warnings, use the following new syntax:
++
+.Example
+[source,yaml]
+----
+ spec:
+   configuration:
+-    restic:
+-      enable: true
++    nodeAgent:
++      enable: true
++      uploaderType: restic
+# ...
+----
+
+[NOTE]
+====
+In the next version of OADP, the Restic option will be deprecated and Kopia will become the default uploaderType.
+====
+
+=== Upgrade steps
+
+==== Upgrade steps from OADP 1.2 Technology Preview Data Mover
+
+OADP 1.2 Data Mover backups *cannot* be restored with OADP 1.3. To prevent a gap in the data protection of your applications, complete the following steps before upgrading to the OADP 1.3 upgrade:
+
+* If your cluster backups are sufficient and Container Storage Interface (CSI) storage is available,
+back up the applications with a CSI backup.
+* If you require off cluster backups:
+.. Back up the applications with a file system backup that uses the `--default-volumes-to-fs-backup=true or backup.spec.defaultVolumesToFsBackup` options.
+.. Back up the applications with your object storage plugins, e.g. velero-plugin-for-aws.
+
+[NOTE]
+====
+If your OADP 1.2 Data Mover backup must be restored, OADP must be fully uninstalled and OADP 1.2 reinstalled and configured.
+====
+
+==== Backing up the DPA configuration
+
+Save your current DataProtectionApplication (DPA) CustomResource configuration and verify that you saved the values.
+
+.Example
+[source,terminal]
+----
+oc get dpa -n openshift-adp -o yaml > dpa.orig.backup
+----
+
+==== Upgrading the OADP Operator
+
+* Change your subscription channel for the OADP Operator from `stable-1.2` to `stable-1.3`.
+* Allow time for the Operator and containers to update and restart.
+
+For general operator upgrade instructions review the https://docs.openshift.com/container-platform/4.14/operators/admin/olm-upgrading-operators.html[OpenShift documentation].
+
+==== Converting DPA to the new version
+
+If you need to move backups off cluster with the Data Mover,  reconfigure the DataProtectionApplication (DPA) as follows:
+
+* You must remove the features.dataMover key and values from the DPA.
+* You must remove the VSM plugin.
++
+.Example
+[source,yaml]
+----
+  spec:
+   configuration:
+-    features:
+-      dataMover:
+-      enable: true
+-      credentialName: dm-credentials
++    nodeAgent:
++      enable: true
++      uploaderType: kopia
+     velero:
+       defaultPlugins:
+-      - vsm
+       - csi
+       - openshift
+# ...
+----
+
+Wait for the DPA to reconcile successfully.
+
+==== Verifying the upgrade
+
+* Follow the instructions to verify the installation. link:https://github.com/weshayutin/oadp-operator/blob/d46d57381a9849dfb791d1bdf564af0582e7e059/docs/install_olm.md#verify-install[Git file]
```

Comment:
same as above, let's write this out in prod doc and link to it.  Do not link to personal git repos in production doc ever :)